codespell_lib/data/dictionary.txt

```diff
@@ -5647,13 +5647,22 @@ deporarily->temporarily
 depracated->deprecated
 depreacte->deprecate
 depreacted->deprecated
+depreacts->deprecates
 deprecatedf->deprecated
 deprectaed->deprecated
 deprectate->deprecate
 deprectated->deprecated
 deprectates->deprecates
 deprectating->deprecating
+deprectation->deprecation
+deprectats->deprecates
```

Comment:
Add the singular version of this as another correction.

Reply:
a8a54f0

Reply:
Sorry, I meant `deprectat->deprecate` as another line, not as a suggestion on the existing correction.

Reply:
@peternewman I'm confused. Instead of clarrifying, do you mind adding it yourself and then merging the commit once it's ready ?

Reply:
Fixed in ebe8681d44453b17e680cddd2f56e09f0abacc59